<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>401</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; color:#000000;&quot;&gt;Vulnerabilities&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(93, 103, 27);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Hackers will try to attack these three &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;pillars of data security in order&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt; to disrupt the a system and flow&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;of data to authorized users&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Integrity_Button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/Imagefiles/orangebutton.png);</string>
    </property>
    <property name="text">
     <string>Integrity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Confidentiality_Button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/Imagefiles/orangebutton.png);</string>
    </property>
    <property name="text">
     <string>Confidentiality</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Availability_Button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/Imagefiles/orangebutton.png);</string>
    </property>
    <property name="text">
     <string>Availability</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back_Button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/Imagefiles/blackbutton.png);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>109</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/Imagefiles/uteplogo.jpeg);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>229</y>
      <width>131</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/Imagefiles/afc.jpeg);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>Navigation</string>
    </property>
    <addaction name="Menu_main"/>
    <addaction name="Menu_tutorial"/>
    <addaction name="Menu_videos"/>
    <addaction name="Menu_attacks"/>
    <addaction name="Menu_counter"/>
    <addaction name="Menu_exit"/>
   </widget>
   <addaction name="menuNavigation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Menu_main">
   <property name="text">
    <string>Main Hub</string>
   </property>
  </action>
  <action name="Menu_tutorial">
   <property name="text">
    <string>Tutorial Menu</string>
   </property>
  </action>
  <action name="Menu_videos">
   <property name="text">
    <string>Videos</string>
   </property>
  </action>
  <action name="Menu_attacks">
   <property name="text">
    <string>Attacks</string>
   </property>
  </action>
  <action name="Menu_counter">
   <property name="text">
    <string>Counter Measures</string>
   </property>
  </action>
  <action name="Menu_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
